md/organization-management-in-bonita-bpm-studio.md

```diff
@@ -74,11 +74,24 @@ Bonita Portal, which could have an impact on deployed processes.
 
 To publish the organization data:
 
-1. Choose **Publish...** from the Organization menu.
-2. Click the organization to be uploaded, and
-then click **_Next_**.
-3. Specify the username and password of the user that will be
+1. Choose **Deploy...** from the Organization menu.
+2. Click the organization to be deployed, and
+3. Specify the username of the user that will be
 used to log in to Bonita Portal when a process is started from
 Bonita Studio. This user must be in the organization you have
 selected.
-4. Click **Publish**.
+4. Click **Deploy**.
+
+## Profile management
+
+In order to log in to bonita UIs (Portal, applications...), a user must have at least on profile (User, Adminstrator...etc).
+
+**_In Community_**, only 3 provided profiles are supported (User, Adminstrator and Process Manager). When deploying an organization from the Studio, each user is automatically mapped to all those profiles.
+
+**_In Teamwork edition_**, those 3 provided profiles can edited in the Studio using an XML editor (Go to Organization > Profiles > Open).
+
+**_In Efficiency and Performance editions_**, profiles can be defined in the Studio using the [Profile Editor](profileCreation.md).
+
```

Comment:
"By default, all users of ACME are mapped to the role _member_, and the role _memeber_ is automatically mapped to all default profiles.
=> also to custom profiles, or only default ones?
Therefore, for testing purposes, you can use any ACME user to login to Bonita Portal and applications.

Reply:
In community only User and Adminstrator profiles exist.
Each users of the an organization (ACME or other) are mapped to those 2 profiles using a user mapping.